Interview/AutomationExercise.md

```diff
@@ -0,0 +1,78 @@
+<p align="center">
+<img src="../logo.png">
+</p>
+
+#Babylon 
+# iOS Automation engineer technical exercise
+
+At babylon, we firmly believe that **transparency** is a core value that should be present in everything we do, and hiring the best person should reflect that belief. That is why we've defined a clear and open technical exercise to be completed by candidates applying for an iOS automation engineer role. 
+
+## Our expectations
+
+Babylon aims to hire only the most qualified people. That doesn't mean that a candidate must have several academic degrees or years of experience. We value **potential** equally as much as **knowledge** and **experience**. Above all, we value one's ability to **think independently**, **solve problems** and to **engage with colleagues**.
+
+As such, these are the main areas the team will focus our assessment of candidates on:
+
+- Technical expertise
+   - Language (Swift) and platform (iOS)
+   - Frameworks (UI testing with XCTest)
+   - Knowledge of software testing practises (Manual and Automated)
+   - Automation Architectural design patterns (Page Object Model, Cucumber)
+   - Knowledge of iOS APIs
+   - Knowledge of the development cycle
+- Interest in sharing one's knowledge
+- Interest in learning new competences
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+## The Technical Exercise
+
+### Application
+
+To begin with please download the following open source [application](https://github.com/khoren93/SwiftHub) and setup your environment. This will demonstrate your understanding of Xcode and building iOS application.
+
+### Existing Tests
+
+The project currently contains a number of UI test cases, for these tests we would like you to complete the following:
+
+- Add a comment to each test outlining what is being tested
+- Describe if you feel the test is appropriate or well written
+- Improve the test if you feel it requires improvement
+- Verify the state of the test
+
+### New Test Cases
+
+For the net stage we would like you to improve test coverage by adding new tests as follows:
+
+- Create new tests to cover the scenarios you feel are more relevant
+- Write an explanation as to why the test is needed and what it covers
```

Comment:
again, I'd leave it for on-site part. The reason is that it's easier to get this explanation verbally (writing good code comments is a good skill but that's not what we are looking at and it might be harder to express this in a comment)